packages/ListBoxBrowser/test/ListBoxBrowser.spec.js

```diff
@@ -0,0 +1,69 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import data from "./fixtures/multiple";
+import dataMultipleFirstOptionNoOptions from "./fixtures/multiple.firstOptionNoOptions";
+import dataNoOptions from "./fixtures/multiple.withNoOptions";
+import { getData, getOptionByKey } from "../src/helpers";
+import ListBoxBrowser from "../src";
+
+const selectedOptions = { current: {} };
+const dataParameters = { data, selectedOptions, defaultSelectedOptions: () => false };
+
+describe("ListBoxBrowser", () => {
```

Comment:
Will you be adding more interactive tests? Most of these tests are just snapshot / see if it renders properly.

Reply:
yeap I going to add cypress, before merging it.